patches/chrome-common-chrome_paths_linux.cc.patch

```diff
@@ -1,28 +1,24 @@
 diff --git a/chrome/common/chrome_paths_linux.cc b/chrome/common/chrome_paths_linux.cc
-index fc47bd3f12418fdaed879e0b485bfc7cb572a6e8..9d7f74be49456b05c593ae2df4831c754d931a73 100644
+index fc47bd3f12418fdaed879e0b485bfc7cb572a6e8..30585ca8b71bb0a7c0e3c817ea15493f4a4212bc 100644
 --- a/chrome/common/chrome_paths_linux.cc
 +++ b/chrome/common/chrome_paths_linux.cc
 @@ -16,6 +16,10 @@
  #include "chrome/common/channel_info.h"
  #include "chrome/common/chrome_paths_internal.h"
  
-+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++#if defined(BRAVE_CHROMIUM_BUILD)
 +#include "brave/common/brave_channel_info_linux.h"
 +#endif
 +
  namespace chrome {
  
  using base::nix::GetXDGDirectory;
-@@ -87,10 +91,18 @@ bool GetDefaultUserDataDirectory(base::FilePath* result) {
+@@ -87,10 +91,14 @@ bool GetDefaultUserDataDirectory(base::FilePath* result) {
```

Comment:
can we do a method override here instead of patch as discussed?

Reply:
Done.